OpenRA.Mods.Common/Traits/BotModules/BaseBuilderBotModule.cs

```diff
@@ -72,6 +72,12 @@ public class BaseBuilderBotModuleInfo : ConditionalTraitInfo
 		[Desc("Number of refineries to build additionally after building a barracks.")]
 		public readonly int AdditionalMinimumRefineryCount = 1;
 
+		[Desc("Minimum distance in cells for resource patches when checking for building placement.")]
+		public readonly int ResourceDistance = 0;
+
+		[Desc("Place the following buildings at least this many cell apart to the next building.")]
+		public readonly Dictionary<string, int> BaseSpacing = null;
```

Comment:
Should this have `[ActorReference(dictionaryReference: LintDictionaryReference.Keys)]`? (and maybe the rest of the file needs some `ActorReference`?)

Reply:
I think this would be a good idea, but it crashes
```
OpenRA.Utility(1,1): Error: OpenRA.Mods.Common.Lint.CheckActorReferences failed with exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at OpenRA.Mods.Common.Lint.LintExts.GetFieldValues(Object ruleInfo, FieldInfo fieldInfo, LintDictionaryReference dictionaryReference) in /home/matthias/Entwicklung/OpenRA/OpenRA.Mods.Common/Lint/LintExts.cs:line 45
   at OpenRA.Mods.Common.Lint.CheckActorReferences.CheckActorReference(Action`1 emitError, ActorInfo actorInfo, TraitInfo traitInfo, FieldInfo fieldInfo, IReadOnlyDictionary`2 dict, ActorReferenceAttribute attribute) in /home/matthias/Entwicklung/OpenRA/OpenRA.Mods.Common/Lint/CheckActorReferences.cs:line 59
   at OpenRA.Mods.Common.Lint.CheckActorReferences.CheckTrait(Action`1 emitError, ActorInfo actorInfo, TraitInfo traitInfo, Ruleset rules) in /home/matthias/Entwicklung/OpenRA/OpenRA.Mods.Common/Lint/CheckActorReferences.cs:line 45
   at OpenRA.Mods.Common.Lint.CheckActorReferences.OpenRA.Mods.Common.Lint.ILintRulesPass.Run(Action`1 emitError, Action`1 emitWarning, ModData modData, Ruleset rules) in /home/matthias/Entwicklung/OpenRA/OpenRA.Mods.Common/Lint/CheckActorReferences.cs:line 28
   at OpenRA.Mods.Common.UtilityCommands.CheckYaml.CheckRules(ModData modData, Ruleset rules, Map map) in /home/matthias/Entwicklung/OpenRA/OpenRA.Mods.Common/UtilityCommands/CheckYaml.cs:line 147
```
so I leave this for another time.